ibm_was/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG - IBM WAS
 
+## 1.13.0 / 2022-01-08
+
+* [Fixed] Add comment to autogenerated model files. See [#10945](https://github.com/DataDog/integrations-core/pull/10945).
+* [Fixed] Remove unused config options. See [#10623](https://github.com/DataDog/integrations-core/pull/10623).
+* [Security] Bump lxml package. See [#10904](https://github.com/DataDog/integrations-core/pull/10904).
```

Comment:
Duplicate entry

Reply:
This is fine since it does affect the dependencies of the check

Reply:
Oh, nvm